data/text/moves.ts

```diff
@@ -2366,7 +2366,7 @@ export const MovesText: {[id: IDEntry]: MoveText} = {
 	},
 	futuresight: {
 		name: "Future Sight",
-		desc: "Deals damage two turns after this move is used. At the end of that turn, the damage is calculated at that time and dealt to the Pokemon at the position the target had when the move was used. If the user is no longer active at the time, damage is calculated based on the user's natural Special Attack stat, types, and level, with no boosts from its held item or Ability. Fails if this move or Doom Desire is already in effect for the target's position.",
+		desc: "Deals damage two turns after this move is used. Damage is calculated and dealt at the end of the second turn to the Pokemon at the original position of the target. If the user is no longer active, damage is calculated based on it's Special Attack stat, types and level ignoring any boosts from held items or ability. Fails if this move or Doom Desire is already in effect for the target's position.",
```

Comment:
If Future Sight is updated, Doom Desire should be updated to match.

You introduce "end of the second turn" in the second sentence when that's already clarified in the first sentence.

You used the wrong "its" for "its Special Attack stat". Using "natural Special Attack stat" also clarifies that it's the Sp. Atk stat used in the summary screen, and I also think it's helpful to clarify that it is in fact the user's Sp. Atk stat, not whatever Pokemon is currently on the field. It also doesn't "ignore" Abilities/items from Pokemon in the back for Future Sight, it just doesn't apply them.